pandas/tests/io/json/test_pandas.py

```diff
@@ -1032,6 +1038,102 @@ def test_to_jsonl(self):
         assert result == expected
         assert_frame_equal(pd.read_json(result, lines=True), df)
 
+    def test_readjson_chunks(self, lines_json_df):
+        """Basic test that read_json(chunks=True) gives the same result as
+        read_json(chunks=False)"""
+        # GH17048: memory usage when lines=True
+
+        for cs in [1, 1.0]:
+
+            unchunked = pd.read_json(StringIO(lines_json_df), lines=True)
+            chunked = pd.concat(
+                pd.read_json(StringIO(lines_json_df), lines=True, chunksize=cs)
+            )
+
+            assert_frame_equal(chunked, unchunked)
+
+    def test_readjson_chunksize_requires_lines(self, lines_json_df):
+        msg = "chunksize can only be passed if lines=True"
+        with tm.assert_raises_regex(ValueError, msg):
+            pd.read_json(StringIO(lines_json_df), lines=False, chunksize=2)
+
+    def test_readjson_chunks_series(self):
+        """Test reading line-format JSON to Series with chunksize param"""
+        s = pd.Series({'A': 1, 'B': 2})
+
+        strio = StringIO(s.to_json(lines=True, orient="records"))
+        unchunked = pd.read_json(strio, lines=True, typ='Series')
+
+        strio = StringIO(s.to_json(lines=True, orient="records"))
+        chunked = pd.concat(pd.read_json(
+            strio, lines=True, typ='Series', chunksize=1
+        ))
+
+        assert_series_equal(chunked, unchunked)
+
+    def test_readjson_each_chunk(self, lines_json_df):
+        """
+        Other tests check that the final result of read_json(chunksize=True) is
```

Comment:
don't use quotes for test use # for comments

Reply:
done